src/osmf/lib/HDSPlayer.as

```diff
@@ -834,7 +837,15 @@ package
     public function onGetCurrentTime(event:Event):void
     {
       var eventObject:Object = new Object();
-      eventObject.currentTime = _mediaPlayerSprite.mediaPlayer.currentTime.toString();
+      if (_playCompleted == true)
+      {
+        eventObject.currentTime = _mediaPlayerSprite.mediaPlayer.duration.toString();
+        _playCompleted = false;
```

Comment:
Hi Angie,

I created new branch PBF-598 from master .I created pull request.

https://github.com/ooyala/html5-video-plugins/pull/144

1)  We created _playCompleted variable to fix this bug(PBF-598). _playCompleted
variable is only get called when videoseek event is get fired followed
by onGetCurrentTime
event. After this event _playCompleted do not play any role so we thought
to reset this variable here(onGetCurrentTime event). But i am agree with
you this is not the right place to reset this variable. So, i removed
it.This will not impact current fix.

2) Previously i created PBF-598 from stable. So, I deleted previous branch
and created new branch from master.

Please review the code and  let me know if there any changes required.

Thanks
Sarita Singh



On Thu, Apr 14, 2016 at 10:45 AM, angieisrani <notifications@github.com>
wrote:

> In src/osmf/lib/HDSPlayer.as
> <https://github.com/ooyala/html5-video-plugins/pull/139#discussion_r59760165>
> :
>
> > @@ -834,7 +837,15 @@ package
> >      public function onGetCurrentTime(event:Event):void
> >      {
> >        var eventObject:Object = new Object();
> > -      eventObject.currentTime = _mediaPlayerSprite.mediaPlayer.currentTime.toString();
> > +      if (_playCompleted == true)
> > +      {
> > +        eventObject.currentTime = _mediaPlayerSprite.mediaPlayer.duration.toString();
> > +        _playCompleted = false;
>
> why set playCompleted to false here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/html5-video-plugins/pull/139/files/bd6487b3cf11b0382bb2bbccde349997cde8ecd6#r59760165>
>



-- 
Best Regards,
Sarita Singh